README.md

```diff
@@ -147,19 +158,24 @@ domainlist relay_to_domains = ${lookup mysql{RELAY_DOMAINS}}
 hostlist   relay_from_hosts = localhost : @ : 192.168.0.0/24
 #trusted_users = www-data
 ```
-Specify here, the username and group under which Exim runs. This combination is also that under which mailman must run in order to work:
+This line configures database connectivity. You need to uncomment it and change the word 'CHANGE', to the password you will use for the 'vexim' database user, which we will set up in the next part. The socket path in Debian is `/var/run/mysqld/mysqld.sock` per default.
 ```
-exim_user = mailnull
-exim_group = mail
+#hide mysql_servers = localhost::(/tmp/mysql.sock)/vexim/vexim/CHANGE
```

Comment:
Should paste the pgsql_servers line here as well.

I'm also thinking perhaps we should either change these lines to make it clear which part of them is the database name, and which is the user name, or add a short comment to the config file right above them.